fhir-term-graph/src/main/java/com/ibm/fhir/term/graph/loader/impl/UMLSTermGraphLoader.java

```diff
@@ -136,8 +132,9 @@ private void loadConcepts() throws FileNotFoundException, IOException {
 
         Map<String, AtomicInteger> sabCounterMap = new HashMap<>();
 
-        try (BufferedReader reader = new BufferedReader(new FileReader(conceptFile))) {
+        try (BufferedReader reader = new BufferedReader(COSObject.getItem(cosBucketName, UMLS_CONCEPT_NAMES_AND_SOURCES_FILE))) {
             reader.lines().forEach(line -> {
+
                 String[] tokens = line.split(UMLS_DELIMITER);
```

Comment:
feels like there should be a check for length

Reply:
Fixed.